### ISSUE TITLE
Native App : On the Home page add pop-up menu option

### ISSUE THREAD

**Issue by abhijeet007rocks8** [DhairyaBahl/React-Messenger-App#85]
On the welcome page after successful login add a header with logo and the three menu dots 
On clicking the three dots a menu pops-up having options to 
---change username 
---change profile pic
---logout 

for this issue work on Native-App branch

**Comment by Yukti-1** [comment]
May I be assigned this task ?

**Comment by abhijeet007rocks8** [comment]
@Yukti-1 Sure you can go ahead work on the Native-App branch
and if you face any issue feel free to contact

**Comment by abhijeet007rocks8** [comment]
@Yukti-1 Updates regarding the issue??

**Comment by Yukti-1** [comment]
> @Yukti-1 Updates regarding the issue??

Yes, I am working on it.

**Comment by lopeselio** [comment]
Is the issue still open ?

**Comment by lopeselio** [comment]
Are there anymore issues open for the native app. I wanna contribute to react native projects for GSSOC21

**Comment by abhijeet007rocks8** [comment]
@lopeselio yes I am creating one for you. Comment one it will be assigned to you.
@Yukti-1 can tell me the status of the issue please.

**Comment by lopeselio** [comment]
One

**Comment by abhijeet007rocks8** [comment]
@lopeselio issue #120  is created with label Native App comment there so that I can assign it you

**Comment by abhijeet007rocks8** [comment]
@Yukti-1 any updates??
if you are facing any issue feel free to discuss with mentors.

**Comment by Yukti-1** [comment]
@abhijeet007rocks8 Ok

**Comment by subhangi2731** [comment]
@abhijeet007rocks8 please assign me this issue
